.gitignore

```diff
@@ -53,9 +26,39 @@ override.tf.json
 *_override.tf.json
 
 # Include override files you do wish to add to version control using negated pattern
-#
 # !example_override.tf
 
 # Include tfplan files to ignore the plan output of command: terraform plan -out=tfplan
 # example: *tfplan*
-**/.sentinel/*
```

Comment:
Was this impacting commiting `.sentinel` files?  From what I remember, this was to stop us from commiting the `.sentinel` directory

Reply:
This is correct.